std/getopt.d

```diff
@@ -1482,3 +1515,30 @@ unittest
         ~ "          This help information.\n";
     assert(wanted == helpMsg, helpMsg ~ wanted);
 }
+
+unittest
+{
+    import std.string : indexOf;
+
+    enum ENUM {
```

Comment:
This must be the only place I've ever seen in Phobos that uses ALL_CAPS identifiers. Could we use `Enum` instead? (The precedent for that is `std.typecons.BitFlags`, which also uses `Enum` in the unittest.)